src/redux/note-details/reducers.ts

```diff
@@ -85,6 +89,22 @@ export const NoteDetailsReducer: Reducer<NoteDetails, NoteDetailsAction> = (
           (action as SetCheckboxInMarkdownContentAction).checked
         )
       }
+    case NoteDetailsActionType.ADD_ALIAS:
+      return {
+        ...state,
+        aliases: state.aliases.concat([(action as AddAliasAction).alias])
+      }
+    case NoteDetailsActionType.REMOVE_ALIAS:
+      return {
+        ...state,
+        aliases: state.aliases.filter((alias) => alias !== (action as RemoveAliasAction).alias),
+        primaryAlias: state.primaryAlias === (action as RemoveAliasAction).alias ? null : state.primaryAlias
+      }
+    case NoteDetailsActionType.SET_PRIMARY_ALIAS:
+      return {
+        ...state,
+        primaryAlias: (action as SetPrimaryAliasAction).alias
+      }
```

Comment:
Add esdoc to the method

Reply:
This would mean adding docs for the whole note-details reducer.
Out of scope.

Reply:
IMHO this is not out of scope because you modified the function.